### PR TITLE
jenkins: disable xdist for tici tests

### PR DESCRIPTION
**Pull request by jnewb1**
pretty sure this is the exception that happened here: https://jenkins.comma.life/blue/organizations/jenkins/openpilot/detail/master/68/pipeline

```
Traceback (most recent call last):
  File "/data/openpilot/selfdrive/modeld/modeld.py", line 319, in <module>
    main(demo=args.demo)
  File "/data/openpilot/selfdrive/modeld/modeld.py", line 122, in main
    config_realtime_process(7, 54)
  File "/data/pythonpath/openpilot/common/realtime.py", line 45, in config_realtime_process
    set_core_affinity(c)
  File "/data/pythonpath/openpilot/common/realtime.py", line 38, in set_core_affinity
    os.sched_setaffinity(0, cores)
OSError: [Errno 22] Invalid argument
```

can replicate about 1/5 times on a 3x. perhaps that cpu is going to sleep or a race condition?

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -25,8 +25,8 @@ export GIT_BRANCH=${env.GIT_BRANCH}
 export GIT_COMMIT=${env.GIT_COMMIT}
 export AZURE_TOKEN='${env.AZURE_TOKEN}'
 export MAPBOX_TOKEN='${env.MAPBOX_TOKEN}'
-# only use 1 thread for tici tests since most require HIL
-export PYTEST_ADDOPTS="-n 0"
+# only use 1 thread for tici tests since most require HIL, and print stdout for debugging
+export PYTEST_ADDOPTS="-n 0 -s"
```
Comment (adeebshihadeh) [inline Jenkinsfile:29]: why isn't it showing once it fails?

**Comment by adeebshihadeh** [comment]
Yes, this is likely a power save race condition in the test (see thermald).

File: Jenkinsfile (hunk 1 above)
Comment (jnewb1) [reply Jenkinsfile:29]: xdist doesn't let you capture stdout, it's very strange. only stderr is actually captured.

Comment (jnewb1) [reply Jenkinsfile:29]: https://github.com/pytest-dev/pytest-xdist/issues/354

Comment (adeebshihadeh) [reply Jenkinsfile:29]: ah, it's the xdist issue. do we use xdist at all on the tici tests? we can just add `-n0`

Comment (jnewb1) [reply Jenkinsfile:29]: it's there

Comment (sshane) [reply Jenkinsfile]: `-n0 -s` seems to be enough to show stdout and stderr on PC from tests, I assume it should be the same on device? I would check before merging this, could be another bug

File: Jenkinsfile [hunk 2]
```diff
@@ -200,7 +200,7 @@ node {
           //["build master-ci", "cd $SOURCE_DIR/release && TARGET_DIR=$TEST_DIR $SOURCE_DIR/scripts/retry.sh ./build_devel.sh"],
           ["build openpilot", "cd selfdrive/manager && ./build.py"],
           ["check dirty", "release/check-dirty.sh"],
-          ["onroad tests", "pytest selfdrive/test/test_onroad.py -s"],
+          ["onroad tests", "pytest selfdrive/test/test_onroad.py"],
```
Comment (sshane) [inline Jenkinsfile:203]:
```suggestion
          ["onroad tests", "pytest -s selfdrive/test/test_onroad.py"],
```

oh, I don't think this works after the test files, you're effectively passing this into unittest

**Comment by adeebshihadeh** [comment]
We should still fix this.

**Pull request closed without merging**